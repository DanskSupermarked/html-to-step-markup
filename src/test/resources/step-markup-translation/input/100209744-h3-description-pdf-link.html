<h3>Vær med på trenden med den spiselige have</h3> <p>Den spiselige have er en populær trend, som danskerne vil bruge endnu mere tid på i fremtiden. I en tid med fokus på det nære, gode råvarer og selvforsyning har køkkenhaven og krydderurterne fået et comeback.     <br /> <br /> Opfyld drømmen om den spiselige have med Plus&rsquo; nye højbede, der kan kombineres, så både blomster, salat og persille kan dyrkes side om side.     <br /> <be>                                                                                                                                 Der er også mulighed for at bygge to ovenpå hinanden i højden, hvis planterne kræver det.<br /> </be></p><h3>Fremstillet i komposit for minimal vedligeholdelse</h3><p>Højbede fremstillet i composit planker giver dig minimal vedligeholdelse, og er et godt alternativ til det trykimprægnerede træ, idet det er et blandingsprodukt med 50% træfiber, 35% genbrugsplast og 15% bindemiddel, uv beskyttelse og farve. Fordelen ved composit er minimal vedligehold og lang holdbarhed. Produktet kræver udelukkende afvaskning med vand, så man slipper for at tænke på slibning og efterbehandling med træbeskyttelse/maling.</p><p><a href="/medias/sys_master/converted/100/209/742/100209742_PLUS_Product_17558_Montage Vejl_178690-1_1.Montage Vejl.original.pdf">Download samlevejledning</a></p> <p>&nbsp;</p> <p>&nbsp;</p>
