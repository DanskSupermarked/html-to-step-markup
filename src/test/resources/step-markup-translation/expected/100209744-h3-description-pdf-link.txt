<H3>Vær med på trenden med den spiselige have</H3>
Den spiselige have er en populær trend, som danskerne vil bruge endnu mere tid på i fremtiden. I en tid med fokus på det nære, gode råvarer og selvforsyning har køkkenhaven og krydderurterne fået et comeback.<return/>
<return/>
Opfyld drømmen om den spiselige have med Plus’ nye højbede, der kan kombineres, så både blomster, salat og persille kan dyrkes side om side.<return/>
Der er også mulighed for at bygge to ovenpå hinanden i højden, hvis planterne kræver det.<return/>
<return/>
<H3>Fremstillet i komposit for minimal vedligeholdelse</H3>
Højbede fremstillet i composit planker giver dig minimal vedligeholdelse, og er et godt alternativ til det trykimprægnerede træ, idet det er et blandingsprodukt med 50% træfiber, 35% genbrugsplast og 15% bindemiddel, uv beskyttelse og farve. Fordelen ved composit er minimal vedligehold og lang holdbarhed. Produktet kræver udelukkende afvaskning med vand, så man slipper for at tænke på slibning og efterbehandling med træbeskyttelse/maling.<return/>
<return/>
Download samlevejledning &lt;/medias/sys_master/converted/100/209/742/100209742_PLUS_Product_17558_Montage Vejl_178690-1_1.Montage Vejl.original.pdf&gt;
